### PR TITLE
[W4.11e][T13-3]Hansel Chia Chen Young

### PR DESCRIPTION
**Pull request by handshou**
Ready for review!

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -10,6 +10,17 @@
 
 public class UtilsTest {
 
+    @Test
+    public void elementsContainNull() throws Exception {
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:14]: It is better to follow the convention that the names of the method being tested and the test method are the same, at least for the sake of better readability and tracking.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -10,6 +10,17 @@
...
+        // contain null objects
+        assertTrue(Utils.isAnyNull((Object) null));
+        assertTrue(Utils.isAnyNull(null, null));
+
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:18]: Maybe test for an empty list as well?

Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:18]: Can add tests for lists with combinations of null and non-null items

**Review by parvathy-sudhir-pillai — COMMENTED** [review]
Overall, good effort @handshou. See my comments for suggestions. Please close this PR after reading comments.
